configure.ac

```diff
@@ -1025,6 +1030,29 @@ if test "x$use_ccache" = "xyes"; then
     AX_CHECK_PREPROC_FLAG([-Qunused-arguments],[CPPFLAGS="-Qunused-arguments $CPPFLAGS"])
 fi
 
+dnl bignum value
+if test x"$req_bignum" = x"auto"; then
+  GMP_CHECK
+  if test x"$has_gmp" = x"yes"; then
+    set_bignum=gmp
+  fi
```

Comment:
Needs changing here to prevent a configure error when libgmp isn't found but `--with-zerocoin-bignum=auto` (the default) is set. the `AC_MSG_WARN` line is optional and may be better suited elsewhere.
```diff
dnl bignum value
if test x"$req_bignum" = x"auto"; then
  GMP_CHECK
  if test x"$has_gmp" = x"yes"; then
    set_bignum=gmp
+ else
+   set_bignum=openssl
+   AC_MSG_WARN([gmp not found, falling back to openssl bignum])
  fi
```

Reply:
BTW, haven't tested this change yet...just looking at the check against `set_bignum` further down